app/jobs/migrate_before_force_delete_statuses_job.rb

```diff
@@ -0,0 +1,9 @@
+class MigrateBeforeForceDeleteStatusesJob < ApplicationJob
+  def perform
+    domains = Domain.where.not(statuses_before_force_delete: nil)
+    domains.each do |domain|
+      domain.force_delete_domain_statuses_history = domain.statuses_before_force_delete
+      domain.save
```

Comment:
How about look for domains not only `.where.not(statuses_before_force_delete: nil)`, but `where(force_delete_domain_statuses_history: nil)`? That could help us keep task idemponent.

Reply:
yes, good idea